pkg/updatestatus/nodeinformer_test.go

```diff
@@ -993,7 +1044,6 @@ func Test_sync_with_node(t *testing.T) {
 					Labels: map[string]string{"node-role.kubernetes.io/some": ""},
 				},
 			},
-			expectedErr: fmt.Errorf("failed to determine which machine config pool the node belongs to: %w", fmt.Errorf("failed to find a matching node selector from 2 machine config pools")),
```

Comment:
We can now update the name of this test case as an error is not expected at the moment.

Reply:
I will keep https://github.com/openshift/cluster-version-operator/pull/1144#discussion_r1938521095 open and close this one as resolved.